plotly_resampler/figure_resampler/figure_resampler_interface.py

```diff
@@ -1291,14 +1291,14 @@ def construct_update_data(
                         )
             # 2.1. Autorange -> do nothing, the autorange will be applied on the
             #      current front-end view
-            elif len(autorange_matches) and not len(spike_matches):
+            elif autorange_matches and not spike_matches:
                 # PreventUpdate returns a 204 status code response on the
                 # relayout post request
                 return dash.no_update
 
         # If we do not have any traces to be updated, we will return an empty
         # request response
-        if len(updated_trace_indices) == 0:
+        if not updated_trace_indices:
```

Comment:
This is for me somewhat less verbose / clear than the `len(...) == 0`

Reply:
What is you opinion on this @jvdd?

Reply:
Is indeed less verbose, but I guess this adjustment is more "Pythonic".
We can make it perhaps more verbose by adding a small inline comment?

Reply:
imo, the surrounding comments make it quite clear that updated_trace_indices should be empty for this if to pass